cmd/utils/flags.go

```diff
@@ -1401,9 +1401,7 @@ func SetP2PConfig(ctx *cli.Context, cfg *p2p.Config, nodeName, datadir string, l
 		cfg.NoDiscovery = true
 	}
 
-	if ctx.IsSet(DiscoveryV5Flag.Name) {
-		cfg.DiscoveryV5 = ctx.Bool(DiscoveryV5Flag.Name)
-	}
+	cfg.DiscoveryV5 = ctx.Bool(DiscoveryV5Flag.Name)
```

Comment:
May I ask why we need this change? I think it does not make any change. By default, `cfg.DiscoveryV5` is false. If `DiscoveryV5Flag` is not set, cfg.DiscoveryV5 will still stay to false.

Reply:
Optimism's spec says discovery v5 should be used by default, so I recently changed it to true: https://github.com/testinprod-io/op-erigon/pull/178